### ISSUE TITLE
Key error: 'testCaseId'

### ISSUE THREAD

**Issue by hellyson-herik** [typhoon-hil/allure-docx#39]
I don't know if I did something wrong in the installation, but this error simply appears when I try to make the call
"allure-docx allure_json report.docx"

**Comment by bjarnesc** [comment]
A test json should contain a testCaseId. What allure version / language are you using?

**Comment by hellyson-herik** [comment]
Python
Allure 2.20.1

**Comment by bjarnesc** [comment]
Are you using the exe? If so instead try installing via pip.

**Comment by hellyson-herik** [comment]
no, i just installed it via pip and tried using the command line

**Comment by bjarnesc** [comment]
Ran a few tests with newest allure-pytest 2.13.1 and everything went fine. You can try on [these example test rsults](https://github.com/typhoon-hil/allure-docx/tree/master/tests/allure-results). If you get the same error something in your system is broken. Else you have to provide more information about what tests you run. Possibly if you have some more advanced pytest/allure techniques. Not every features were tested and are safe to work.

Edit: Checkout https://github.com/typhoon-hil/allure-docx/blob/master/.github/issue_template.md 
Hard to help if so little information is provided.

**Comment by hellyson-herik** [comment]
understood, in the case I use behave, will that be the difference? I will compare the results of allure and if I see something I will comment again, thanks for the help

**Comment by feixu88** [comment]
Have you solved the problem? I also meeting the same problem.

**Comment by feixu88** [comment]
> Ran a few tests with newest allure-pytest 2.13.1 and everything went fine. You can try on [these example test rsults](https://github.com/typhoon-hil/allure-docx/tree/master/tests/allure-results). If you get the same error something in your system is broken. Else you have to provide more information about what tests you run. Possibly if you have some more advanced pytest/allure techniques. Not every features were tested and are safe to work.
> 
> Edit: Checkout https://github.com/typhoon-hil/allure-docx/blob/master/.github/issue_template.md Hard to help if so little information is provided.

I have tried 'example test results'，it is ok.However,I meeting the same error 'Key error: 'testCaseId' ' for my test results.

**Comment by hellyson-herik** [comment]
> Have you solved the problem? I also meeting the same problem.


I couldn't solve it, I ended up leaving it aside

**Comment by bjarnesc** [comment]
> > Ran a few tests with newest allure-pytest 2.13.1 and everything went fine. You can try on [these example test rsults](https://github.com/typhoon-hil/allure-docx/tree/master/tests/allure-results). If you get the same error something in your system is broken. Else you have to provide more information about what tests you run. Possibly if you have some more advanced pytest/allure techniques. Not every features were tested and are safe to work.
> > Edit: Checkout https://github.com/typhoon-hil/allure-docx/blob/master/.github/issue_template.md Hard to help if so little information is provided.
> 
> I have tried 'example test results'，it is ok.However,I meeting the same error 'Key error: 'testCaseId' ' for my test results.

Did you work with allure-behave also? I'm not experienced with behave so unfortunatly I cannot tell what leads to the error but if you use it too it is likely that this is the problem, as behave was not tested.

**Comment by ggros** [comment]
I have the same issue, and problem is probably quite simple, testCaseId is optional attribute, my tests are not decorated with this attribute

https://allurereport.org/docs/gettingstarted-readability/

https://allurereport.org/docs/junit5-reference/#id

**Comment by bjarnesc** [comment]
> I have the same issue, and problem is probably quite simple, testCaseId is optional attribute, my tests are not decorated with this attribute
> 
> https://allurereport.org/docs/gettingstarted-readability/
> 
> https://allurereport.org/docs/junit5-reference/#id

See https://github.com/typhoon-hil/allure-docx/issues/44#issuecomment-1752824705 . I fixed the issue in my fork (see merge request)
